### PR TITLE
#477 unmarshalling exception handling in ccj import

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/download/CcjImportDownloadReaderTest.java [hunk 1]
```diff
@@ -50,15 +51,22 @@ public void before() {
     }
     
     
+    //------------------------ PRIVATE --------------------------
```
Comment (pawelczak) [inline saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/download/CcjImportDownloadReaderTest.java]: TESTS code section

File: saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/download/CcjImportDownloadReaderTest.java [hunk 2]
```diff
@@ -137,22 +159,57 @@ public void read_findJudgmentIds_FOUND() throws Exception {
...
+        DateTime publicationDate = new DateTime(2013, 03, 03, 12, 30);
+        readerOpen(publicationDate);
+        
+        SourceCcJudgmentTextData textData1 = createCcjTextData("abc1");
+        SourceCcJudgmentTextData textData2 = createCcjTextData("abc1a");
+        
+        List<String> judgmentIds = Lists.newArrayList("35", "38", "40");
+        when(sourceCcjExternalRepository.findJudgmentIds(eq(0), eq(pageSize), eq(publicationDate))).thenReturn(judgmentIds);
+        
+        when(sourceCcjExternalRepository.findJudgment(eq("35"))).thenReturn(textData1);
```
Comment (pawelczak) [inline saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/download/CcjImportDownloadReaderTest.java]: Is is neccesary to use Mockito macher eq here?

File: saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/download/CcjImportDownloadReaderTest.java (hunk 1 above)
Comment (madryk) [inline saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/download/CcjImportDownloadReaderTest.java]: :+1:

File: saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/download/CcjImportDownloadReaderTest.java (hunk 2 above)
Comment (madryk) [inline saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/download/CcjImportDownloadReaderTest.java]: It's not necessary.

Comment (pawelczak) [inline saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/download/CcjImportDownloadReaderTest.java]: Ok, I was just wondering.

**Pull request closed without merging**